README.md

```diff
@@ -99,6 +99,39 @@ of an Eclipse Hono instance as shown in the diagram below.
    back end component which receives vehicle data via Hono's north bound Kafka based Telemetry API and writes it to the
    Influx DB.
 
+
+# Using Zenoh Router to geographically distribute the vehicle data
+
+The integration of Eclipse Zenoh into the FMS blueprint is planned to be realized in two steps:
+
+**Step 1:  Extend FMS Forwarder to publish data in Zenoh and configure Zenoh Router to also receive data over MQTT**
+
+- Assess the current FMS Forwarder and identify the necessary modifications to enable data publication through Zenoh.
+- Implement the required code changes to publish data via Zenoh.
+- MQTT plugin configuration on the Zenoh Router to as well support MQTT interfaces.
+
+**Step 2: FMS consumer Integration in Zenoh Router**
+
+- Implement the necessary changes in Zenoh Router using Zenoh Flow to create components that publish / consume data into influxdb.
+
+Currently, step 1 has been implemented, and we now provide the instructions on how you can test the communication using Zenoh, and we are actively advancing with the ongoing work on step 2.
+
+<img src="img/architecture-zenoh.drawio.png">
+
+1. Create a local docker image for FMS Forworder to support the zenoh communication, run the below command from  fleet-management/components/ directory:
+   
+   ```sh
+   docker build -t fms-forwarder:custom -f Dockerfile.fms-forwarder .
+   ```
+   
+2. Start up the zenoh router and back end services using Docker Compose files in the top level (fleet-management/) folder:
+
+   ```sh
+   docker compose -f ./fms-blueprint-compose.yaml -f ./fms-blueprint-compose-zenoh.yaml up --detach
+   ```
+
+Here you find the [Zenoh based Python Subscriber](https://github.com/vivekpandey02/fms-zenoh-test-subscriber) application to validate the data transmissted by FMS Fowarder.
+
```

Comment:
please only describe the status quo established by this PR. If you extend/amend the functionality in subsequent PRs, you can amend/change the text accordingly.

Reply:
Yes, modified it.